addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -228,3 +228,56 @@ img.o_forum_avatar_big {
 .website_forum {
     margin-bottom: $spacer;
 }
+
+.forums main {
+    background: $gray-100;
+}
+.o_forum_card {
+    overflow: hidden;
+    padding-top: 56%;
+    border: none;
+    box-shadow: 0 1px 2px gray('400');
+}
+.o_forum_card_header {
+    background-color: $white;
+    position: absolute;
+    top: 0;
+    z-index: 1;
+    width: 100%;
+    h1 {
+        margin: 0;
+        color: $gray-800;
+        font-weight: bold;
+        line-height: 1.2rem;
+        font-size: 14px;
+
+    }
+}
+.o_forum_card_img {
+    background: linear-gradient(99deg, theme-color('secondary') 10%, darken(theme-color('secondary'), 10%) 90%);
```

Comment:
The background is not covered by the white background 

You can see on the right picture the strange corner from o_forum_card_header

![image](https://user-images.githubusercontent.com/4848797/58485390-41abc680-8164-11e9-9973-377d3b0ac4be.png)